eldritch/mythos.py

```diff
@@ -412,6 +498,129 @@ def get_trigger(self, event, state):
+    return events.BinaryChoice(
+        event.character,
+        "Shop at estate sale?", "Yes", "No",
+        events.Purchase(event.character, "unique", draw_count=2, keep_count=2),
+        events.Nothing(),
+    )
+
+
+class Mythos17(Environment):
+  def __init__(self):
+    super().__init__(
+        "Mythos17", "WitchHouse", "Cave",
+        {"plus"}, {"moon"},
+        "urban", activity_location="University"
+    )
+
+  def get_trigger(self, event, state):
+    if not isinstance(event, events.Movement):
+      return None
+    if event.character.place.name != self.activity_location:
+      return None
+    check = events.Check(event.character, "lore", -1)
+    gain = events.Gain(event.character, {"clues": 1})
+    return events.PassFail(event.character, check, gain, events.Nothing())
+
+
+class Mythos18(Environment):
+  def __init__(self):
+    super().__init__("Mythos18", "Square", "Unnamable", {"plus"}, {"moon"}, "mystic")
+
+  def get_interrupt(self, event, state):
+    if (
+        isinstance(event, events.GainOrLoss)
+        and values.Calculation(event.gains.get("sanity", 0), operand=operator.gt, right=0).value(
+            state)
+        and (event.source is None or event.source.name not in ("Psychology", "Asylum"))
+    ):
+      return events.GainPrevention(self, event, "sanity", event.gains["sanity"])
+    return None
+
+
+class Mythos19(CloseLocationsHeadline):
+  def __init__(self):
+    super().__init__(
+        "Mythos19", "WitchHouse", "Cave", {"moon"}, {"plus"}, ["Merchant"], evict=False
+    )
+
+
+class Mythos20(ReturnMonstersHeadline):
+  def __init__(self):
+    super().__init__(
+        "Mythos20", "Cave", "Roadhouse", {"moon"}, {"plus"}, from_places=["streets"]
+    )
+
+
+class Mythos21(Headline):
+  def __init__(self):
+    super().__init__("Mythos21", "Graveyard", "Isle", {"moon"}, {"plus"})
+
+  def create_event(self, state):
+    seq = super().create_event(state)
+    seq.events.append(events.IncreaseTerror())
+    return seq
+
+
+class Mythos22(ReturnAndIncreaseHeadline):
+  def __init__(self):
+    super().__init__(
+        "Mythos22", "Square", "Unnamable", {"hex"}, {"slash", "triangle", "star"}, ["Tentacle Tree"]
+    )
+
+
+class Mythos23(ReleaseMonstersHeadline):
+  def __init__(self):
+    super().__init__("Mythos23", "Unnamable", "Woods", {"moon"}, {"plus"}, to_places="Merchant")
+
+
+class Mythos24(ReturnMonstersHeadline):
+  def __init__(self):
+    super().__init__(
+        "Mythos24", "WitchHouse", "Cave", {"circle"}, {"square", "diamond"},
+        from_places=["Easttown", "Roadhouse", "Diner", "Police"]
+    )
+
+
+class Mythos25(CityBonus):
+  def __init__(self):
+    super().__init__(
+        "Mythos25", "Woods", "Square", {"square", "diamond"}, {"circle"}, "weather",
+        bonus_skill="lore", penalty_skill="fight",
+    )
+
+  def get_modifier(self, thing, attribute, state):
+    # FAQ p10: If the phrase ___ is not within the text of the effect, then it affects investigators
```

Comment:
Is this where you wanted this comment?

Reply:
I guess here _and_ in 44 would make sense... it's not obvious that a heat wave or heavy rain in Massachussetts should affect what's happening in Antartica or Pluto.